src/components/DayPicker.jsx

```diff
@@ -55,6 +58,9 @@ const defaultProps = {
   orientation: HORIZONTAL_ORIENTATION,
   withPortal: false,
   hidden: false,
+  navPrev: null,
+  navNext: null,
+  useNavDefaultStyles: true,
```

Comment:
Should the `DateRangePicker` or `SingleDatePicker` ever have the ability to change this or should this remain kind of like a hidden prop?